src/oemof_tabular_plugins/wefe/facades/pv_panel.py

```diff
@@ -8,6 +8,9 @@
 
 from oemof.tabular._facade import Facade, dataclass_facade
 
+from oemof_tabular_plugins.wefe.facades import functions as f
```

Comment:
this is a bit dangerous to use such a generic and short alias, I would rather import the needed functions

Reply:
Besides it seems you are not using those functions, no?